config/routes.rb

```diff
@@ -1,20 +1,39 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
-      resources :tests, only: %i[index]
-
+      get "role_users/create"
```

Comment:
[must]
createやdestroyなのにGETメソッドなのは変
```suggestion
      resources: :role_users, only: %i[create, destroy]
```

Reply:
これは理解です